data/_models/shared/Federated data access.csv

```diff
@@ -0,0 +1,11 @@
+tableName,tableExtends,tableType,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,refLabel,defaultValue,validation,visible,computed,semantics,profiles,label,description
+Federated data access,,,,,,,,,,,,,,,,,"DataCatalogueFlat,FederatedDataAccess",,List of Armadillo and/or Opal server(s) available for this resource.
+Federated data access,,,resource,ref,1,TRUE,,Resources,,,,,,,,,"DataCatalogueFlat,FederatedDataAccess",,Federated data access defined for this resource
```

Comment:
use refSchema 'catalogue' and refTable 'Resources'. Do not import and fill out Resources table in Federated access staging area.